website/docs/style/design-tokens/design-tokens.md

```diff
@@ -96,54 +96,34 @@ const App = DesignTokens;
-
-Let's take a look at the two main theme creation situations: local and global.
+**We recommend you using the native Figma variables (tokens) functionality**. But if you need more functionality that Figma doesn't have at the moment, use [Tokens Studio plugin for Figma](https://www.figma.com/community/plugin/843461159747178978). It's one of the most powerful tools for managing tokens, linking styles between the code and Figma files.
 
-#### **Local theme: you need to try on other tokens or a new theme for a component or part of the design system**
+Refer to the following tutorials, for the detailed process for creating a new theme:
 
-In this case we recommend you using [Tokens Studio plugin for Figma](https://www.figma.com/community/plugin/843461159747178978). It's one of the most powerful tools for managing tokens, linking styles between the code and Figma files, and it can help you save time trying new values for tokens of your new theme.
+- Internal libraries have two tutorials, [tutorial for Figma tokens](https://www.figma.com/design/1TV7YbEL3FaV0znCkQtsrC/Themes'-playground-%26-tutorial-%F0%9F%8E%93?node-id=13125-73031&node-type=canvas&t=qOTf0DSn0M8p63of-11) and [tutorial for Tokens Studio](https://www.figma.com/design/K1s6wF8NTH3uNHvjkn6hjc/Themes'-playground-%26-tutorial-%F0%9F%8E%93?m=auto&t=jHrLhhOMB32IMklB-6)
+- Public libraries now have only [tutorial for Tokens Studio](https://www.figma.com/community/file/1274028958101796491/semrush-design-tokens)
 
 ::: tip
-There, you can find the detailed process for creating a new theme for our design system explained: [internal](https://www.figma.com/file/K1s6wF8NTH3uNHvjkn6hjc/Themes-playground-%26-tutorial-%F0%9F%8E%93?node-id=24%3A90461&t=uZCoQy8xPBjC1ctm-11), [public](https://www.figma.com/community/file/1274028958101796491/semrush-design-tokens).
-:::
-
-#### **Global theme: you need to develop a theme for the entire design system**
-
-In this case, you need to take more steps.
+**In cases where different styles are needed for just one component or a part of the design system:**
 
-1. Duplicate [library with tokens](/get-started-guide/work-figma/work-figma#core_libraries).
-2. Using the [Tokens Studio plugin for Figma](https://www.figma.com/community/plugin/843461159747178978), you connect the tokens of the default theme with your dublicated file. How to use it to create your own set of tokens for the new theme, check the playground: [internal](https://www.figma.com/file/K1s6wF8NTH3uNHvjkn6hjc/Themes-playground-%26-tutorial-%F0%9F%8E%93?node-id=24%3A90461&t=uZCoQy8xPBjC1ctm-11), [public](https://www.figma.com/community/file/1274028958101796491/semrush-design-tokens).
-3. Test and try on the theme for all components of the library in the theme playground.
-4. Save the JSON of the theme, and either give it to the developers, or commit it yourself.
-5. Voila! You are awesome!
+- Designer can follow the same scenario described above
```

Comment:
```suggestion
- Designer can create a new theme as described earlier, and apply it only to the necessary component
```
I was a bit confused by this sentence and started thinking what scenario exactly you mean, for some reason thinking that you might mean something from the beginning of the page... Maybe it will be a bit clearer with this suggestion?

Reply:
Yes, this will be definitely better 👍